README.md

```diff
@@ -1 +1,78 @@
-# codacy-test
\ No newline at end of file
+# Codacy ESLint
+
+## Adding new plugins / configs
+
+1.  Install the package using npm
+2.  \[Plugins only\] Add the plugin to the plugins section in the file `src/eslintDefaultOptions.ts`
+3.  \[Plugins only\] If the plugin has descriptions for rules on GitHub, reference them
+at `src/docGeneratorMain.ts` to include them on the generated documentation. To do this, add a section similar to the following example:
+
+    ```typescript
+    console.log("Generate xss description files")
+    await docGenerator.downloadDocs(
+      (pattern) =>
+        `${githubBaseUrl}/Rantanen/eslint-plugin-xss/master/docs/rules/${pattern}.md`,
+      "xss"
+    )
+    ```
+
+4.  Generate documentation so it adds the new plugin documentation
+
+## Generating documentation
+
+```bash
+npm install
+npm run generateDocs
+```
+
+## Test changes to codacy-seed locally
+You may need to test changes that comes from our (codacy-engine-typescript-seed)[https://github.com/codacy/codacy-engine-typescript-seed].
+
+1.  Create a package with your changes on the seed:
+    - Don't forget to update the dependencies: `npm install`
+    - Compile the library: `npm run compile`
+    - Package the library: `npm pack`
+        > This should generate a codacy-seed-0.0.1.tgz on your codacy-seed repository
+
+2.  Copy the `codacy-seed-0.0.1.tgz` into the root of this repository
+
+3.  Install the package: `npm install codacy-seed-0.0.1.tgz`
+
+4.  Update Dockerfile and `.dockerignore` so you copy the `codacy-seed-0.0.1.tgz` inside the docker you will be building
+    *  Add `!codacy-seed-0.0.1.tgz` to your `.dockerignore`
+     *  Add the package to the docker before `RUN npm install`: `COPY codacy-seed-0.0.1.tgz ./`
+   *  Remove multi-stage docker steps
+        *  Lines from `FROM node:$NODE_IMAGE_VERSION` to `RUN rm -rf /package.json /package-lock.json`
+        > This way you skip copying the files to the other docker, and another `npm install`
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion
5. Publish your docker locally as normal: `docker build -t codacy-eslint:local .`
```